doc/noobaa-crd.md

```diff
@@ -240,6 +240,22 @@ spec:
   dbConf: |+
     max_connections = 1000
 ```
+## Pod Topology Spread Constraints for Noobaa endpoint
+
+Pod Topology Spread Constraints is enabled by default in Noobaa.
```

Comment:
This short sentence is not clear enough to me. I would explain that the operator will set this on the noobaa endpoint deployment in order to force spreading endpoint pods evenly between nodes, and that this will not work before kubernetes 1.26 so the operator checks for it, and also skips if this is already set on the deployment, or the CR annotation is set. I would also explain that users can make changes to this after the operator creates it and it will not override it.